CMakeLists.txt

```diff
@@ -17,7 +17,10 @@ if(BUILD_GREENTEA_TESTS)
 endif()
 
 if(${CMAKE_CROSSCOMPILING})
-    include(${MBED_CONFIG_PATH}/mbed_config.cmake)
+    if(NOT DEFINED MBED_CONFIG_INCLUDE)
+        include(${MBED_CONFIG_PATH}/mbed_config.cmake)
```

Comment:
I think this guard should be part of `mbed_config.cmake`

Reply:
It could be but this requires changes in the mbed-tools :) mbed_config.cmake is generated file

Reply:
We should be able to use https://cmake.org/cmake/help/latest/command/include_guard.html. This could be added to https://github.com/ARMmbed/mbed-tools/blob/master/src/mbed_tools/build/_internal/templates/mbed_config.tmpl. Should be a simple change. I can raise a PR tomorrow if you don't feel like raising a new one.

Reply:
Sounds good to me. Yes, please create the PR with this change. I hope it can be merged quickly :)